en/docs/deploy/security/configure-transport-level-security.md

```diff
@@ -139,13 +139,14 @@ name in the `deployment.toml` file.
 2.  Add a new server name by adding the following
     property (under the relevant Tomcat connector configuration):
 
-    ``` toml
+    ```
     [transport.https.properties]
     server="WSO2 WSO2 IS server"
-    
```

Comment:
why is this line removed?

Reply:
Because having the empty line in between made the code snippet appear in a much larger font. To resolve that, the line was removed